kernel/mmio.c

```diff
@@ -19,52 +19,129 @@
  ******************************************************************************/
 
 #include "mmio.h"
-#include <stdio.h>
 
 uint8_t
 read8(uint32_t address)
-{
-	return (uint8_t) (read32(address-(address % 4)) >> ((address % 4) * 8));
-}
+  {
```

Comment:
I think those brackets are indented one to far. To me it seems like they don't have to be indented, according to Andreas' style

Reply:
You're right, the method-brackets are not indented as in `for`, `while`, ...
I corrected that.